inspirehep/modules/records/api.py

```diff
@@ -610,7 +610,13 @@ def _query_citing_records(self, show_duplicates=False, session=None):
         citation_query = session.query(RecordMetadata).with_entities(RecordMetadata.id,
                                                                      RecordMetadata.json['control_number'])
         citation_filter = referenced_records(RecordMetadata.json).contains([ref])
-        citations = citation_query.filter(citation_filter)
+        filter_deleted_records = or_(not_(type_coerce(RecordMetadata.json, JSONB).has_key('deleted')),  # noqa: W601
+                                     not_(RecordMetadata.json['deleted'] == cast(True, JSONB)))
+        only_literature_collection = type_coerce(RecordMetadata.json['_collections'],
```

Comment:
you can do this in a simpler way:
https://github.com/inspirehep/inspire-next/blob/4d116e13e625dc4ea4a8b9a3ab2bcf6261ecec77/inspirehep/modules/disambiguation/core/db/readers.py#L85-L86